src/odemis/driver/autoscript_client.py

```diff
@@ -1770,12 +1795,31 @@ def terminate(self):
             self._pos_poll.cancel()
             self._pos_poll = None
 
+    def _get_coordinate_system_offset(self):
```

Comment:
By convention, `get_*()` functions return a value. Here, the function only store the value. So let's pick a different name. For instance, `_store_coordinate_system_offset()`, or `_update_coordinate_system_offset()`.